pom.xml

```diff
@@ -877,6 +877,12 @@
                             <version>${jbossLoggingProcessorVersion}</version>
                             <scope>compile</scope>
                         </dependency>
+                        <dependency>
+                            <groupId>javax.annotation</groupId>
+                            <artifactId>jsr250-api</artifactId>
+                            <version>1.0</version>
```

Comment:
Feel free to push it but I was thinking that it would be better to have it in a variable with the other versions so that we don't forget to update if one day there is a new version. Might be purely hypothetical though.